src/Console/Command/SetupCICommand.php

```diff
@@ -42,29 +42,62 @@ protected function execute(InputInterface $input, OutputInterface $output): int
-                $this->symfonyStyle->warning('The "rector.yaml" workflow already exists');
+        if ($ci !== CiDetector::CI_GITHUB_ACTIONS) {
+            $noteMessage = sprintf(
+                'Only Github Action is supported for now.%sCreate an issue to add your CI %s',
+                PHP_EOL,
+                'https://github.com/rectorphp/rector/issues/'
+            );
+
+            $this->symfonyStyle->note($noteMessage);
+            return self::SUCCESS;
+        }
+
+        $rectorWorkflowFilePath = getcwd() . '/.github/workflows/rector.yaml';
+        if (file_exists($rectorWorkflowFilePath)) {
+            $response = $this->symfonyStyle->ask('The "rector.yaml" workflow already exists. Overwrite it?', 'Yes');
+            if (! in_array($response, ['y', 'yes', 'Yes'], true)) {
+                $this->symfonyStyle->note('Nothing changed');
                 return self::SUCCESS;
             }
+        }
 
-            $currentRepository = $this->resolveCurrentRepositoryName(getcwd());
-            if ($currentRepository === null) {
-                $this->symfonyStyle->error('Current repository name could not be resolved');
+        $currentRepository = $this->resolveCurrentRepositoryName(getcwd());
+        if ($currentRepository === null) {
+            $this->symfonyStyle->error('Current repository name could not be resolved');
 
-                return self::FAILURE;
-            }
+            return self::FAILURE;
+        }
 
-            $workflowTemplate = FileSystem::read(__DIR__ . '/../../../templates/rector-github-action-check.yaml');
+        $workflowTemplate = FileSystem::read(__DIR__ . '/../../../templates/rector-github-action-check.yaml');
 
-            $workflowContents = strtr($workflowTemplate, [
-                '__CURRENT_REPOSITORY__' => $currentRepository,
-            ]);
+        $workflowContents = strtr($workflowTemplate, [
+            '__CURRENT_REPOSITORY__' => $currentRepository,
+        ]);
 
-            FileSystem::write($rectorWorkflowFilePath, $workflowContents);
-            $this->symfonyStyle->success('The "rector.yaml" workflow was added');
-        }
+        FileSystem::write($rectorWorkflowFilePath, $workflowContents);
+
+        $this->symfonyStyle->newLine();
+
+        $this->symfonyStyle->note('The "rector.yaml" workflow was added');
+        $this->symfonyStyle->newLine();
+
+        $this->symfonyStyle->title('2 steps more to run you Github Action:');
+        $this->symfonyStyle->writeln(
+            '1) Generate new Github Token here:' .
+            PHP_EOL .
+            'https://github.com/settings/tokens/new'
+        );
+
+        $this->symfonyStyle->newLine();
+
+        $this->symfonyStyle->writeln(
+            '2) Add it to your repository secrets under "GITHUB_TOKE" name:'
+            . PHP_EOL .
+           sprintf('https://github.com/%s/settings/secrets/actions/new', $currentRepository)
+        );
+
+        $this->symfonyStyle->newLine();
```

Comment:
**💭 thought:** If it is possible to add some links to a documentation page:
* Fine grained Organization access token: https://docs.github.com/en/organizations/managing-programmatic-access-to-your-organization/setting-a-personal-access-token-policy-for-your-organization
* Fine grained user access token: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-fine-grained-personal-access-token

Reply:
I see. Most people don't read these, so we have limit for 2 links maximum. This command should help people just dump it and use it.

Which 2 links would you go for having typical Rector developer in mind?

Reply:
I think the best would be an info message at the end, telling "If you want to get deeper and configure actions for a whole organization, please see [...]"